minecraft_servers/gamerherz/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+  "server_name": "Gamerherz",
+  "nice_name": "Gamerherz.eu",
+  "direct_ip": "Gamerherz.eu",
+  "server_wildcards": [
+    "%.gamerherz.eu"
+  ],
+  "supported_languages": [
+     "de",
+     "en"
+  ],
+  "social": {
+    "web": "https://www.gamerherz.eu/",
```

Comment:
Website offline

Reply:
Is back online in 2 or 3 days

Reply:
Yeah, and today it's back online. It was planned to be online today and now it's back online